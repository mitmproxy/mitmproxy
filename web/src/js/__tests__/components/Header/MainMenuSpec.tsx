import * as React from "react";
import FlowListMenu from "../../../components/Header/FlowListMenu";
import { render } from "../../test-utils";

test("MainMenu", () => {
    const { asFragment } = render(<FlowListMenu />);
    expect(asFragment()).toMatchSnapshot();
});
