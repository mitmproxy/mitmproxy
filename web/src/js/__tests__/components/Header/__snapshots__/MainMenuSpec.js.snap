// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenu Component should render and connect to state 1`] = `
<div
  className="main-menu"
>
  <div
    className="menu-group"
  >
    <div
      className="menu-content"
    >
      <div
        className="filter-input input-group"
      >
        <span
          className="input-group-addon"
        >
          <i
            className="fa fa-fw fa-search"
            style={
              Object {
                "color": "black",
              }
            }
          />
        </span>
        <input
          className="form-control"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Search"
          type="text"
          value="~u foo"
        />
      </div>
      <div
        className="filter-input input-group"
      >
        <span
          className="input-group-addon"
        >
          <i
            className="fa fa-fw fa-tag"
            style={
              Object {
                "color": "hsl(48, 100%, 50%)",
              }
            }
          />
        </span>
        <input
          className="form-control"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Highlight"
          type="text"
          value="~a bar"
        />
      </div>
    </div>
    <div
      className="menu-legend"
    >
      Find
    </div>
  </div>
  <div
    className="menu-group"
  >
    <div
      className="menu-content"
    >
      <div
        className="filter-input input-group"
      >
        <span
          className="input-group-addon"
        >
          <i
            className="fa fa-fw fa-pause"
            style={
              Object {
                "color": "hsl(208, 56%, 53%)",
              }
            }
          />
        </span>
        <input
          className="form-control"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Intercept"
          type="text"
          value=""
        />
      </div>
      <button
        className="btn-sm btn btn-default"
        onClick={[Function]}
        title="[a]ccept all"
      >
        <i
          className="fa fa-fw fa-forward text-success"
        />
        Resume All
      </button>
    </div>
    <div
      className="menu-legend"
    >
      Intercept
    </div>
  </div>
</div>
`;
